collector/compile-benchmarks/README.md

```diff
@@ -15,13 +15,15 @@ There are three categories of compile-time benchmarks, **Primary**, **Secondary*
 These are real programs that are important in some way, and worth tracking.
 They mostly consist of real-world crates.
 
-- **bitmaps-3.1.0**: A bitmaps implementation. Stresses code handling traits.
-- **cargo-0.60.0**: The Rust package manager. A large program, and an important
-  one in the Rust ecosystem.
-- **clap-3.1.6**: A command line argument parser. A crate used by many Rust
-  programs.
+- **bitmaps-3.1.0**: A bitmaps implementation. Stresses the compiler's trait
+  handling by implementing a trait `Bits` for the type `BitsImpl<N>` for every
+  `N` value from 1 to 1024.
+- **cargo-0.60.0**: The Rust package manager. A large program, a binary crate,
```

Comment:
We only benchmark the [library](https://github.com/rust-lang/rustc-perf/blob/73903101b833c2b0cced8cb4ce1c7ab4f8d22a45/collector/compile-benchmarks/cargo-0.60.0/perf-config.json#L2) part and not the binary part.

Reply:
Good catch, thanks.